### PR TITLE
Update the readme file with environment setup instructions

### LINKED ISSUE #117: Update the readme file with environment setup instructions
**Is your feature request related to a problem? Please describe.**
Currently, the readme file does not have setup instructions for the project. 
 
**Describe the solution you'd like**
Update the readme file with environment setup instructions. (only the frontend)

### PR DESCRIPTION
**Pull request by Bawanthathilan**
## Purpose

The purpose of this PR is to fix #117 

## Goals

Update the readme file with environment setup instructions

### Screenshots

image

### CODE REVIEW

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks @Bawanthathilan , I added a comment.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
@Bawanthathilan, the correct readme should be the readme file of the root directory. 

Please delete `donor-portal/README.md` and `admin-portal/README.md` files

**Comment by Bawanthathilan** [comment]
> @Bawanthathilan, the correct readme should be the readme file of the root directory.
> 
> Please delete `donor-portal/README.md` and `admin-portal/README.md` files

sure

File: README.md [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+  ```
+
+- Visit your app at http://localhost:3000
 
 ## Build
 
 Please make sure you have all the listed prerequisites tools installed locally.
 
 1. Node - Latest
 2. Ballerina - Latest
```
Comment (jayasanka-sack) [inline README.md:50]: No longer needed as we moved backend to separate repos. 
```suggestion
```

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Bawanthathilan !

**Pull request merged**